disco_aws_automation/disco_alarm_config.py

```diff
@@ -168,13 +173,14 @@ class DiscoAlarmsConfig(object):
     Represenation of all alarms in config file.
     """
 
-    def __init__(self, environment, config_file=None):
+    def __init__(self, environment, config_file=None, autoscale=None):
         if config_file:
             self.config = ConfigParser()
             self.config.read(config_file)
         else:
             self.config = read_config(DEFAULT_CONFIG_FILE)
         self.environment = environment
+        self._autoscale = autoscale or None  # laziliy intialized
```

Comment:
It probably is unnecessary, but it seems to be the pattern we've settled on, so not sure we need to change it.